examples/test_world.py

```diff
@@ -0,0 +1,7 @@
+from pathlib import Path
+
+from mat_dp_pipeline import Pipeline
+
+pipeline = Pipeline()
+
+output = pipeline(Path("./World"))
```

Comment:
This doesn't do much. What's the intention of this file? As I described in the other comment - to always run it with pytest (sanity check that nothing crashes)? Or more of a documentation? If it's the latter, then this should be put to a docstring instead of code of test_world.py imho.

Reply:
I use examples to test print values typically - pytest can be a pain for this as if it runs 10 tests you can't tell which sent the print etc. So e.g. I add a print in pipeline for testing, I need something to run to show it. They are also useful for more complex examples, but for now I see these are not complex

Reply:
Similarly pdb.set_trace() - pytest doesn't like that one bit

Reply:
I debug with pytest all the time. No problem there.

Reply:
In fact, it's very easy to run or debug a single test, out of the IDE (or not from IDE for those who don't like it)

Reply:
I'd be quite interested to see your method for this! It's always been quite a bind for me so if you know a better way I'm all ears

Reply:
Nothing special, all built-in in the VS ;) Happy to show you.